flatten-data-ghcn_d.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<p>Would a <code>requirements.txt</code>  file be helpful?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>